hreuniversalis/events/GER_Events.txt

```diff
@@ -732,31 +732,45 @@ country_event = {
 	}
 
 	option = {
-		name = flavor_ger.9.westphalian
```

Comment:
"westphalian" goes by "hannoverian" and is already listed as one of the first (vanilla) options